packages/terra-form-input/src/terra-dev-site/test/form-input/SupportedTypesInputField.test.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import InputField from '../../../InputField';
+import styles from './common/Input.test.module.scss';
+
+const cx = classNames.bind(styles);
+
+const supportedTypes = [undefined, 'text', 'number', 'password', 'email', 'search', 'tel', 'url', 'hidden'];
+
+class SupportedTypesInputField extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isInvalid: false,
+      isIncomplete: false,
+      required: false,
+    };
+
+    this.toggleInvalid = this.toggleInvalid.bind(this);
+    this.toggleIncomplete = this.toggleIncomplete.bind(this);
+  }
+
+  createInputFieldTypeExamples(arr) {
+    const inputFieldTypeExamples = [];
+    for (let i = 0; i < arr.length; i += 1) {
+      inputFieldTypeExamples.push((
+        <div className={cx('content-wrapper')}>
+          <InputField
+            inputId={`unsupported-${arr[i]}-input`}
+            error="Error message."
+            help="Help message."
+            isInvalid={this.state.isInvalid}
+            isIncomplete={this.state.isIncomplete}
+            required={this.state.required}
+            inputAttrs={{
+              name: `supported-${arr[i]}-input`,
+              type: arr[i],
+              ariaLabel: `Input type=${arr[i]}`,
```

Comment:
Our field implementations already provide for proper accessibility connecting the inputs and labels, so this become redundant, and we don't want to suggest consumers should also provide an `aria-label` on the inputs.

```diff
-              ariaLabel: `Input type=${arr[i]}`,
```

Reply:
updated here https://github.com/cerner/terra-core/pull/3386/commits/451995dda70a0c87b88b1a69e7e802692ae84c2d